### ISSUE TITLE
intake.gui AttributeError

### ISSUE THREAD

**Issue by aaronspring** [intake/intake#563]
with intake==0.6.0 `import intake; intake.gui` works as documented: https://github.com/intake/intake/blob/master/docs/source/gui.rst

with current master intake==0.6.0+95.g308b69a, it fails:
```
intake.gui
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-b19b214fa5de> in <module>
----> 1 intake.gui

AttributeError: module 'intake' has no attribute 'gui'

Tried to search for PRs changing gui but didnt find.

**Comment by martindurant** [comment]
The change (I can dig up the PR) was made to remove the ambiguity between the instance of the GUI and the subpackage in which it is defined.

The subpackage is:
```
import intake.interface
```
and the instance is a simple attribute
```
intake.gui
assert isinstance(intake.gui, intake.interface.gui.GUI)
```

(note that the top level init is special, and attributes like this cause imports to happen - this is on purpose, to decrease the time `import intake` takes)

**Comment by aaronspring** [comment]
I dont understand your explanation. importing intake.interface works, but I dont find gui
```
import intake.interface
dir(intake.interface)
['LooseVersion',
 '__builtins__',
 '__cached__',
 '__doc__',
 '__file__',
 '__getattr__',
 '__loader__',
 '__name__',
 '__package__',
 '__path__',
 '__spec__',
 'do_import',
 'gl',
 'output_notebook']
```

let me rephrase the issue: I want to use the functionality of `intake.gui` as in intake==0.6.0. How do I open the GUI panel?

**Comment by martindurant** [comment]
I guess something is going wrong in instantiating the object.
There is a logger
```
import intake
intake.logger
```
which you can configure before attempting
```
intake.gui
```
and it will tell you at least the import that it's trying to make, and presumably failing.
I believe for `gui`, it's the equivalent of
```
import intake.interface
intake.interface.instance
```

**Comment by aaronspring** [comment]
works now with `intake=0.6.3`
